tasks/components/layout/property/PropertyDetailsField.tsx

```diff
@@ -0,0 +1,141 @@
+import { Scrollbars } from 'react-custom-scrollbars-2'
+import { Input } from '@helpwave/common/components/user-input/Input'
+import type { PropertyFieldType } from '@/components/layout/property/property'
+import { fieldTypeList } from '@/components/layout/property/property'
+
+type PropertyDetailsFieldTranslation = {
+  field: string,
+  fieldType: string,
+  values: string,
+  multiSelect: string,
+  singleSelect: string,
+  number: string,
+  text: string,
+  date: string,
+  allowCustomValues: string,
+  allowCustomValuesDescription: string,
+  newEntry: string
+}
+
+const defaultPropertyDetailsFieldTranslation: Record<Languages, PropertyDetailsFieldTranslation> = {
+  en: {
+    field: 'Field',
+    fieldType: 'Field Type',
+    values: 'Values',
+    multiSelect: 'Multi Select',
+    singleSelect: 'Single Select',
+    number: 'Number',
+    text: 'Text',
+    date: 'Date',
+    allowCustomValues: 'Allow custom values',
+    allowCustomValuesDescription: 'Let users enter a free text when the predefined values are not enough.',
+    newEntry: 'New Entry'
+  },
+  de: {
+    field: 'Property Eingabe', // TODO better translation
+    fieldType: 'Property Typ',
+    values: 'Werte',
+    multiSelect: 'Multi Select',
+    singleSelect: 'Single Select',
+    number: 'Zahl',
+    text: 'Text',
+    date: 'Datum',
+    allowCustomValues: 'Hinzufügen neuer Werte',
+    allowCustomValuesDescription: 'Werte können neu hinzugefügt werden,wenn sie nicht vorhanden sind.',
+    newEntry: 'Neuer Eintrag'
+  }
+}
+
+export type PropertyDetailsFieldProps = {
+  value: PropertyFieldType,
+  onChange: (value: PropertyFieldType) => void
+}
+
+/**
+ * The Layout for the PropertyDetails basic information input
+ */
+export const PropertyDetailsField = ({
+  language,
+  value,
+  onChange
+}: PropsWithLanguage<PropertyDetailsFieldTranslation, PropertyDetailsFieldProps>) => {
+  const translation = useTranslation(language, defaultPropertyDetailsFieldTranslation)
+  return (
+    <InputGroup title={translation.field}>
+      <Select
+        // TODO add icons
+        value={value.fieldType}
+        label={{ name: translation.fieldType, labelType: 'labelMedium' }}
+        options={fieldTypeList.map(fieldType => ({ value: fieldType, label: translation[fieldType] }))}
+        onChange={fieldType => onChange({ ...value, fieldType })}
+      />
+      {(value.fieldType === 'multiSelect' || value.fieldType === 'singleSelect') && (
+        <div className={tw('flex flex-col mt-2 gap-y-1')}>
+          <div className={tw('flex flex-row justify-between items-center')}>
+            <Span type="labelMedium">{translation.values}</Span>
+            <Plus
+              className={tw('text-white bg-hw-primary-400 hover:text-gray-100 hover:bg-hw-primary-600 rounded-full mr-3')}
+              size={20}
+              onClick={() => {
+                const newList = [...value.entryList]
+                newList.push(translation.newEntry)
```

Comment:
`translation.newEntry` is not a correct value to push. If you want to display `New entry`, use `placeholder="..."`. An empty value in this list should really be empty/null.